phing/build.xml

```diff
@@ -1,4 +1,5 @@
 <project name="blt" default="list" phingVersion="2.8.2">
+  <property environment="env"/>
```

Comment:
Agreed, I thought this too - but it wasn't available to me during testing unless I added this. According to the docs you need to add this if you want to access `env` https://www.phing.info/trac/ticket/952